docker-compose.yml

```diff
@@ -0,0 +1,105 @@
+services:
```

Comment:
Could we create a folder in the repo and move all the sh and yml files there? 
Or in leave them in .devcontainer folder?

Reply:
The docker-compose and setup files can be used, even if you're not using devcontainers, but I'm fine moving them into a separate folder as well. I think in that cast, I'd leave them in .devcontainer